lib/tasks/generate-from-blueprint.js

```diff
@@ -12,6 +14,19 @@ class GenerateTask extends Task {
   }
 
   async run(options) {
+    let fileInfos = await this.createBlueprints(options);
+
+    if (!options.skipLintFix) {
+      try {
+        let fileDisplayPaths = fileInfos.map(({ displayPath }) => displayPath);
+        await lintFix.run(fileDisplayPaths);
+      } catch (error) {
+        logger.error('Lint fix failed');
```

Comment:
I think we should include a little more context about the error (maybe interpolate `error.message`)

Reply:
done